bindings/cs/MongoDB.Crypt.Example/MongoDB.Crypt.Example.csproj

```diff
@@ -6,7 +6,7 @@
     <TargetFrameworks Condition="'$(BuildingInsideVisualStudio)' != 'true'">net452;netcoreapp2.1</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1</TargetFrameworks>
 
-    <Platforms>x64</Platforms>
+    <Platforms>AnyCPU</Platforms>
```

Comment:
This line fixes the bitness warning the driver gets when consuming the wrapper

Reply:
We're not really supporting AnyCpu right?

Any attempt to run this wrapper on a 32-bit platform is now a runtime exception?

Reply:
Correct!